assets/static/source.go

```diff
@@ -15,17 +16,17 @@ import (
 // StaticSource is an asset source which loads assets from a static JSON file
 type StaticSource struct {
 	s struct {
-		Channels    []*types.Channel           `json:"channels" validate:"omitempty,dive"`
-		Classifiers []*types.Classifier        `json:"classifiers" validate:"omitempty,dive"`
-		Fields      []*types.Field             `json:"fields" validate:"omitempty,dive"`
-		Flows       []*types.Flow              `json:"flows" validate:"omitempty,dive"`
-		Globals     []*types.Global            `json:"globals" validate:"omitempty,dive"`
-		Groups      []*types.Group             `json:"groups" validate:"omitempty,dive"`
-		Labels      []*types.Label             `json:"labels" validate:"omitempty,dive"`
-		Locations   []*utils.LocationHierarchy `json:"locations"`
-		Resthooks   []*types.Resthook          `json:"resthooks" validate:"omitempty,dive"`
-		Templates   []*types.Template          `json:"templates" validate:"omitempty,dive"`
-		Ticketers   []*types.Ticketer          `json:"ticketers" validate:"omitempty,dive"`
+		Channels    []*types.Channel          `json:"channels" validate:"omitempty,dive"`
+		Classifiers []*types.Classifier       `json:"classifiers" validate:"omitempty,dive"`
+		Fields      []*types.Field            `json:"fields" validate:"omitempty,dive"`
+		Flows       []*types.Flow             `json:"flows" validate:"omitempty,dive"`
+		Globals     []*types.Global           `json:"globals" validate:"omitempty,dive"`
+		Groups      []*types.Group            `json:"groups" validate:"omitempty,dive"`
+		Labels      []*types.Label            `json:"labels" validate:"omitempty,dive"`
+		Locations   []*envs.LocationHierarchy `json:"locations"`
```

Comment:
locations have never really been assets.. and this is making more want to ditch that idea.. but for now it still works